js/scripts.js

```diff
@@ -1,145 +1,163 @@
-						detailsUrl: item.url
-					};
-					add(pokemon);
-					console.log(pokemon); //just for debbuging
-				});
-			})
-			.catch((e) => {
-				// hidingMessage();
-				console.error(e);
-			});
-	};
+  //fetch data from api
+  let loadList = () => {
+    // loadingMessage();
+    return fetch(apiUrl)
+      .then((response) => {
+        // hidingMessage();
+        if (!response.ok)
+          throw new Error(`Status Code Error: ${response.status}`);
+        return response.json();
+      })
+      .then((data) => {
+        data.results.forEach((item) => {
+          let pokemon = {
+            name: item.name.toUpperCase(),
+            detailsUrl: item.url,
+          };
+          add(pokemon);
+          console.log(pokemon); //just for debbuging
+        });
+      })
+      .catch((e) => {
+        // hidingMessage();
+        console.error(e);
+      });
+  };
 
-	//function for fetching the details from detailsUrl
-	let loadDetails = (pokemon) => {
-		// loadingMessage();
-		let nextUrl = pokemon.detailsUrl;
-		return fetch(nextUrl)
-			.then((response) => {
-				// hidingMessage();
-				if (!response.ok) throw new Error(`Status Code Error: ${response.status}`);
-				return response.json();
-			})
-			.then((data) => {
-				pokemon.id = data.id;
-				pokemon.imgUrl = data.sprites.front_default;
-				pokemon.height = data.height / 10;
-				pokemon.weight = data.weight / 10;
-				pokemon.abilities = data.abilities.map((ability) => ability.ability.name).join(', ');
-				pokemon.types = data.types.map((type) => type.type.name).join(', ');
-			})
-			.catch((e) => {
-				//hidingMessage();
-				console.error(e);
-			});
-	};
+  //function for fetching the details from detailsUrl
+  let loadDetails = (pokemon) => {
+    // loadingMessage();
+    let nextUrl = pokemon.detailsUrl;
+    return fetch(nextUrl)
+      .then((response) => {
+        // hidingMessage();
+        if (!response.ok)
+          throw new Error(`Status Code Error: ${response.status}`);
+        return response.json();
+      })
+      .then((data) => {
+        pokemon.id = data.id;
+        pokemon.imgUrl = data.sprites.front_default;
+        pokemon.height = data.height / 10;
+        pokemon.weight = data.weight / 10;
+        pokemon.abilities = data.abilities
+          .map((ability) => ability.ability.name)
+          .join(", ");
+        pokemon.types = data.types.map((type) => type.type.name).join(", ");
+      })
+      .catch((e) => {
+        //hidingMessage();
+        console.error(e);
+      });
+  };
 
-	function showModal(item) {
-		let modalBody = $('.modal-body');
-		let modalTitle = $('.modal-title');
-		let modalHeader = $('.modal-header');
+  function showModal(item) {
+    var h = $(".modal").height() / 2;
+    $(".modal").css({
+      display: "flex",
+      top: "50%",
+      left: "50%",
+      position: "absolute",
+      "margin-left": "-150px",
+      "margin-top": "-" + h + "px",
+      width: "30rem",
+      height: "40rem",
+    });
```

Comment:
- Added this piece of code to center the modal. Since you're using jQuery with JavaScript, I just added a jQuery code to override the bootstrap.